mindmeld/components/nlp.py

```diff
@@ -108,6 +116,7 @@ def __init__(self, app_path, resource_loader=None, config=None):
         self.resource_loader = resource_loader or ResourceLoader.create_resource_loader(
             app_path
         )
+        self.language, self.locale = get_language_config(app_path)
```

Comment:
This class is also subclassed by DomainProcessor, IntentProcessor, EntityProcessor. Is it intended that each of these classes also need to know about language and locale?

Reply:
Yes, this needs to be done since the client is directly invoke the process api for the DomainProcessor, IntentProcessor, EntityProcessor.